lib/manageiq/rpm_build/generate_gemset.rb

```diff
@@ -0,0 +1,141 @@
+require 'awesome_spawn'
+require 'fileutils'
+require 'pathname'
+require 'yaml'
+
+module ManageIQ
+  module RPMBuild
+    class GemSet
```

Comment:
Minor, but the class name should match the path name (so either rename the file or rename the class.

Reply:
(Oh just saw you already noted this in the OP)